documentation/project-docs/SDK-PR-guide.md

```diff
@@ -0,0 +1,56 @@
+# SDK PR Guide
+
+## Branching
+See the [versions](https://learn.microsoft.com/en-us/dotnet/core/porting/versioning-sdk-msbuild-vs#lifecycle) document for more details on how SDK versions map to MSBuild and Visual Studio.
+
+### Major releases
+The .NET SDK has yearly major releases aligning with the .NET Runtime in November.  These releases will be of the form N.0.1xx. The feature work for major releases will be done in `main`. 
+Typically, previews for each major release start in February of each year.  For each preview, we will snap a new branch off of main approximately midway through the prior month.
+For .NET 7 for example, the preview branches were created typically around the 21st of each month prior with the later previews forking earlier to provide more time for stabilization and codeflow.
+Main branch has moved to be the next year's release branch in August.
+
+### Minor releases
+The .NET SDK has quarterly minor releases that align with Visual Studio releases. These SDK releases are open to all bug fixes and some features work as well. 
+These quarterly releases should avoid breaking changes if at all possible. Breaking changes instead should be included in the major releases and be tied to customer TargetFramework updates when possible.
+
+Minor releases lock down after preview 3 for each release.  Preview 3 is typically 1 month prior to GA and locks down by the end of the month before the preview release.
+
+#### Transition from minor release to servicing
+After the minor release locks down, it transitions into servicing mode. This typically happens around the preview 3 release to allow time for any late public changes we want feedback on. The next release after preview 3 is typically GA and done as a servicing release. The following steps are done when transitioning from minor release previews to servicing
+- [ ] Follow the new branch check list for creating a new minor release (steps listed on internal OneNote)
+- [ ] Set stabilize branding to true
+- [ ] Enable internal runtime darc subscriptions and trigger them
+
+### Servicing releases
+The .NET SDK has monthly servicing releases aligning with the .NET Runtime servicing releases. These are for top fixes and security updates only to limit risk.
```

Comment:
```suggestion
The .NET SDK has monthly servicing releases (e.g.`7.0.100` VS `7.0.101`) aligning with the .NET Runtime servicing releases. These are for top fixes and security updates only to limit risk.
```

Reply:
Not sure if this suggestion is factually true... I don't recall seeing a `7.0.111`. for example.  Also, we may want to mention why there is no `7.1.00`, for example. This sometimes trips me up.

Reply:
This is accurate. We haven't gotten to 7.0.111 yet. We're on 6.0.114 now since it's been over a year since 6.0 shipped.